{
    "data": {
        "@version": "3.0",
        "@time": "2024-07-19 08:49:13",
        "@type": "post",
        "@filename": "b000000010.xml",
        "open_time": "0000-00-00",
        "close_time": "0000-00-00",
        "lang_name": "Big5",
        "board_id": "1001305431",
        "node": "000000010",
        "site": "1000110001",
        "pt": "2023-09-28 20:12:45",
        "poster": "11127207",
        "realname": "楊竣傑",
        "email": "jacky115526@gmail.com",
        "homepage": null,
        "subject": "108.若遞迴到某層就異常終止，要如何強制它繼續執行到下一層？",
        "content": "<p>可以使用 try-catch 塊來捕獲異常，然後讓遞迴繼續。</p>\r\n\r\n<p>像是在遞迴過程中執行錯誤時 用throw std::runtime_error(&quot;異常訊息&quot;); 這段程式來避免異常 , 這樣即使出現異常程式依然可以跑到下一層程式碼</p>",
        "attach": null,
        "rcount": null,
        "rank": null,
        "hit": "25",
        "lang": "1"
    }
}